port/esp32/hawkbit/hawkbit_certificates.h

```diff
@@ -0,0 +1,50 @@
+/****************************************************************************
+ *
+ * Copyright (c) 2023 plgd.dev s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"),
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef HAWKBIT_CERTIFICATES_H
+#define HAWKBIT_CERTIFICATES_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static const char hawkbit_server_certificate[] =
+  "-----BEGIN CERTIFICATE-----\n"
+  "MIIC0TCCAbkCFDIWFaqBvFlwHGRjh/WMTRcrPotrMA0GCSqGSIb3DQEBCwUAMCUx\n"
+  "CzAJBgNVBAYTAlNLMRYwFAYDVQQDDA0xOTIuMTY4LjAuMTY1MB4XDTIzMTIwNzA5\n"
+  "Mjc1OFoXDTI0MTIwNjA5Mjc1OFowJTELMAkGA1UEBhMCU0sxFjAUBgNVBAMMDTE5\n"
+  "Mi4xNjguMC4xNjUwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDHKaRK\n"
+  "BvmFz7lkrc9BMix4yxOxjSeObNhBtFgjhY1+uLGQ25/y9ZHDx2y976Vp7Ebl+LUI\n"
+  "fURsrj4gt+HtA0hkLudIyPmR5urJF+pUl8s6DkvNyD5nyZVmx5XLjmVxYLeYz52a\n"
+  "f2IlOSxPGgbl+JprolJIo8w9jDjbkB7iHTpCNz2Q75osGYBD4pT0K8co+SVPV0Hh\n"
+  "GyzHuSFNmyMsScb+i+EoGraq4THlUfG1DgPiWJEhR8uOlVBQVLbuMCAp+VbzAhS8\n"
+  "lCn8O9YatCUHB6vGSG6o4SYiHoKovu5dJYCf7KFhKl2xTuG68DxIgZIz2HDB+ZWV\n"
+  "5thCw/wweBBtAUtJAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAEdM3PcNLJk22A9R\n"
+  "5gHm1yB5j09akDLN9eLNT74zOmLtNZ+CBR4nxu44bvzpovLJOL2umVdH24Dv3wiv\n"
+  "7t/EQmC6JRUE9ffWjG9No1gyETK5NLCGaCvCrf/D5hppP5k8Pwtrv/gAejc5SDNU\n"
+  "QjifAnP55owgeyKIWaF/9Tk48nt6HWTZETivGVnJiyiCAHhdKN1adcZ/zj90mPia\n"
+  "JOORaEW0mWA+U+3t20I8Nk5BL0aDgIBwzFXF2FcxX2H9IK/FGVmIWV8HtNqGLU2L\n"
+  "7CUVzJFBEnwdbElF5U3lBRkvup/gIQWwhN8ZoVrLuClSNjPnd2h5+XF3P8QoQGGZ\n"
+  "DkuRToQ=\n"
+  "-----END CERTIFICATE-----\n"
```

Comment:
not needed -> cert will be provisioned from OBT tool / client-application to iotivity-lite cred resource.

Reply:
removed